gubernator.go

```diff
@@ -283,7 +285,29 @@ func (s *Instance) GetPeerRateLimits(ctx context.Context, r *GetPeerRateLimitsRe
 
 // HealthCheck Returns the health of our instance.
 func (s *Instance) HealthCheck(ctx context.Context, r *HealthCheckReq) (*HealthCheckResp, error) {
+	var errs []string
+
 	s.peerMutex.RLock()
+
+	// Iterate through peers and get their last errors
+	peers := s.conf.LocalPicker.Peers()
```

Comment:
I think we should add a method on the `RegionPeerPicker` interface called `Peers() []*PeerClient` implemented like this

```golang
func (rp *RegionPicker) Peers() []*PeerClient {
	var peers []*PeerClient
	for _, peer := range rp.Peers() {
		if peerInfo := rp.GetByPeerInfo(peer.info); peerInfo == nil {
			peers = append(peers, peer)
		}
	}
	return peers
}
```

So we can include the regional peers here like this
```golang
peers := s.conf.LocalPicker.Peers()
peers = append(peers, s.conf.RegionPicker.Peers()...)
```

We can use this same method to collect the shutdown peers at gubernator.go:371